### PR TITLE
Examples fixes3.0.0

### PR DESCRIPTION
**Pull request by victorwiebe**
Checking in some fixes for the examples. We are going to fix all examples next week so everything is working. In the meanwhile we can merge in some changes. Some comments have been included to help fix the GTM related code later on

### CODE REVIEW

File: examples/etherDividend.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import { BigNumber } from '@0x/utils';
 import { RedundantSubprovider, RPCSubprovider, Web3ProviderEngine } from '@0x/subproviders';
-import { Web3Wrapper } from '@0x/web3-wrapper';
-import { EtherDividendCheckpointEvents, ModuleFactoryContract } from '@polymathnetwork/abi-wrappers/lib/src';
+import { EtherDividendCheckpointEvents } from '@polymathnetwork/abi-wrappers/lib/src';
```
Comment (monitz87) [inline examples/etherDividend.ts]: This should be imported from the package root

File: examples/generalPermissionManager.ts [hunk 2]
```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
   await generalPM.addDelegate({ delegate: myAddress, details: 'details' });
 
   const generalTMAddress = (await tickerSecurityTokenInstance.getModulesByName({
-    moduleName: ModuleName.generalTransferManager,
+    moduleName: ModuleName.GeneralTransferManager,
   }))[0];
 
   // Get all delegates
-  await generalPM.changePermission({ valid: true, perm: 'FLAGS', delegate: myAddress, module: generalTMAddress });
+  await generalPM.changePermission({ valid: true, perm: 'ADMIN', delegate: myAddress, module: generalTMAddress });
```
Comment (monitz87) [inline examples/generalPermissionManager.ts]: Why isn't the `Perm` enum being used here?

File: examples/generalPermissionManager.ts [hunk 3]
```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
...
 
   // Check  delegate
   console.log('Delegate is added:');
   console.log(await generalPM.checkDelegate({ delegate: myAddress }));
   console.log('Delegate has flags perm added on general transfer manager:');
   console.log(await generalPM.checkPermission({ delegate: myAddress, module: generalTMAddress, permission: 'FLAGS' }));
```
Comment (monitz87) [inline examples/generalPermissionManager.ts]: These console.log statements should change to reflect the change in permission used

File: examples/generalPermissionManager.ts [hunk 4]
```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
...
   const generalTM = await polymathAPI.moduleFactory.getModuleInstance({
-    name: ModuleName.generalTransferManager,
+    name: ModuleName.GeneralTransferManager,
     address: generalTMAddress,
   });
   await generalTM.changeAllowAllWhitelistTransfers({ allowAllWhitelistTransfers: true });
 
   // Revoking Permission
-  await generalPM.changePermission({ valid: false, perm: 'FLAGS', delegate: myAddress, module: generalTMAddress });
+  await generalPM.changePermission({ valid: false, perm: 'ADMIN', delegate: myAddress, module: generalTMAddress });
```
Comment (monitz87) [inline examples/generalPermissionManager.ts]: Why isn't the Perm enum being used here?

File: examples/generalPermissionManager.ts [hunk 5]
```diff
@@ -129,27 +128,35 @@ window.addEventListener('load', async () => {
...
 
   console.log('Delegate perm has been revoked. Check permission result: ');
   console.log(await generalPM.checkPermission({ delegate: myAddress, module: generalTMAddress, permission: 'FLAGS' }));
```
Comment (monitz87) [inline examples/generalPermissionManager.ts]: These console.log statements should change to reflect the change in permission used

File: examples/etherDividend.ts (hunk 1 above)
Comment (victorwiebe) [reply examples/etherDividend.ts]: There were a few imports mistakes like this. I have fixed them. I am going to move on to the perms task for the next while, seems there are many to update

File: src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper.ts [hunk 6]
```diff
@@ -106,7 +111,7 @@ interface PermissonsPerModule {
   /** Module address */
```
Comment (monitz87) [inline src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper.ts]: This interface's name has a typo, it should be `PermissionsPerModule`

File: src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper.ts [hunk 7]
```diff
@@ -236,7 +241,7 @@ export default class GeneralPermissionManagerWrapper extends ModuleWrapper {
       (value, key): void => {
         const permissonsPerModule: PermissonsPerModule = {
```
Comment (monitz87) [inline src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper.ts]: This variable name has a typo, should be `permissionsPerModule`

File: src/utils/convert.ts [hunk 8]
```diff
@@ -94,3 +94,13 @@ export function parsePartitionBytes32Value(value: string): Partition {
       throw new Error('Partition not recognized');
   }
 }
+export function parsePermBytes32Value(value: string): Perm {
+  switch (bytes32ToString(value)) {
+    case 'ADMIN':
```
Comment (monitz87) [inline src/utils/convert.ts]: Here the case values can be the permission values as well

i.e. `case Perm.Admin`

File: src/utils/convert.ts [hunk 9]
```diff
@@ -94,3 +94,13 @@ export function parsePartitionBytes32Value(value: string): Partition {
...
+      return Perm.Admin;
+    case 'OPERATOR':
```
Comment (monitz87) [inline src/utils/convert.ts]: Here the case values can be the permission values as well

i.e. `case Perm.Operator`

File: src/utils/convert.ts [hunk 10]
```diff
@@ -94,3 +94,13 @@ export function parsePartitionBytes32Value(value: string): Partition {
...
+      return Perm.Operator;
+    default:
+      throw new Error('Partition not recognized');
```
Comment (monitz87) [inline src/utils/convert.ts]: Error message should be `Permission not recognized`

**Comment by victorwiebe** [comment]
Merging example fixes. Yarn Jest and Start are now working fine

**Pull request merged**